api/src/main/java/ca/bc/gov/educ/api/batchgraduation/listener/DistributionRunYearlyCompletionNotificationListener.java

```diff
@@ -105,12 +103,12 @@ protected boolean processGlobalList(DistributionSummaryDTO summaryDTO, String se
 			schoolDistributionPrintFile(studentList,batchId,usl,mapDist);
 		});
 		if (!cList.isEmpty()) {
-			callDistribution = true;
 			DistributionRequest distributionRequest = DistributionRequest.builder().mapDist(mapDist).activityCode(activityCode).studentSearchRequest(getStudentSearchRequest(searchRequest)).build();
 			distributionRequest.setSchools(summaryDTO.getSchools());
-			restUtils.mergeAndUpload(batchId, distributionRequest, activityCode, "N");
+			DistributionResponse response = restUtils.mergeAndUpload(batchId, distributionRequest, activityCode, "N");
+			return response != null && FAILED.name().equalsIgnoreCase(response.getMergeProcessResponse()) ? false : true;
 		}
-		return callDistribution;
+		return false;
```

Comment:
@sumathi-thirumani just wondering if it is always a failed run when cList is empty